pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -1,10 +1,12 @@
 """Class implementing TES API-server-side controller methods."""
 
+import logging
 from copy import deepcopy
 from datetime import datetime
-import logging
```

Comment:
Not sure why you changed the order here, it was fine as it was - alphabetically _by module_, not by keyword.

That is:

```
import module1  # comes first because module1 < module2, even though import > from
from module2 import something
```

Reply:
I was using [isort ](https://pycqa.github.io/isort/) for sorting the imports, if you want it the way it was before I will fix them in the next commit.

Reply:
Ah okay. Fine to use `isort`, but the important thing is consistency. So either use the currently used style in your proposed changes OR change it according to `isort` conventions in _all_ modules (but then that should be done in a separate PR). Anyway, leaving this now, it's not _that_ important.